release-content/0.15/release-notes/14880_Initial_implementation_of_the_Bevy_Remote_Protocol_Adopted.md

```diff
@@ -1,4 +1,65 @@
-<!-- Initial implementation of the Bevy Remote Protocol (Adopted) -->
-<!-- https://github.com/bevyengine/bevy/pull/14880 -->
+We have introduced the Bevy Remote Protocol, which allows the ECS of a running
+Bevy application to be interacted with remotely. This can be used, for example,
+to inspect and edit entities and their components at runtime. We anticipate 
+that this will be used primarily to create things like inspectors for editing
+and debugging.
 
-<!-- TODO -->
+The default methods included with the associated plugin include:
+- getting the serialized values of a set of components from an entity;
+- performing a query for all entities matching a set of components and retrieving
+  their associated values;
+- creating a new entity with a given set of component values;
+- despawning an entity;
+- inserting a set of components into an entity;
+- removing a set of components from an entity;
+- reparenting one or more entities;
+- listing the components registered in the ECS or present on an entity.
+
+Details on these methods are available in the `bevy_remote` module documentation.
+Additionally, custom methods can also be written.
```

Comment:
I don't think this is helpful without more explanation. Either explain, link to docs or cut.